### PR TITLE
Feature/generic webhook

### PR DESCRIPTION
**Pull request by thomaslacher**
First draft for the generic template of a webhook receiver.

### CODE REVIEW

**Comment by thomaslacher** [comment]
Yes that works for me !
I'll make the changes !

File: main.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import (
+	"encoding/json"
+	"github.com/prometheus/alertmanager/template"
+	"log"
+	"net/http"
+	"os"
+)
+
+type JsonResponse struct {
```
Comment (louisfelix) [inline main.go]: My go-lint tells me 
- "exported type JsonResponse should have comment or be unexported". 
- "type JsonResponse should be JSONResponse"
I agree.
```suggestion
// Webhook http response
type JSONResponse struct {
```

**Comment by thomaslacher** [comment]
Done !

File: main.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	listenAddress := ":9876"
+	if os.Getenv("PORT") != "" {
+		listenAddress = ":" + os.Getenv("PORT")
+	}
+
+	log.Printf("listening on: %v", listenAddress)
+	log.Fatal(http.ListenAndServe(listenAddress, nil))
+}
+
+func sendJsonResponse(w http.ResponseWriter, status int, message string) {
```
Comment (louisfelix) [inline main.go]: My go-lint tells me:
- "func sendJsonResponse should be sendJSONResponse"
```suggestion
func sendJSONResponse(w http.ResponseWriter, status int, message string) {
```

**Comment by thomaslacher** [comment]
Updated

**Pull request merged**